UnleashClient/periodic_tasks/fetch_and_load.py

```diff
@@ -18,9 +20,10 @@ def fetch_and_load_features(
     strategy_mapping: dict,
     request_timeout: int,
     request_retries: int,
+    engine: UnleashEngine,
     project: Optional[str] = None,
 ) -> None:
-    (feature_provisioning, etag) = get_feature_toggles(
+    (feature_toggle_json, etag) = get_feature_toggles(
```

Comment:
Bit unsure about the name, especially now that it's a str and not a dict. We're not lying, it's still a JSON, but in a string. In other places we used `state`, maybe that could work better here?

Reply:
Cheers, good suggestion!